src/test/scenarios/managed-network/output/src/managed-network/azext_managed_network/generated/_params.py

```diff
@@ -22,7 +22,6 @@
 def load_arguments(self, _):
 
     with self.argument_context('managed-network managed-network list') as c:
-        c.argument('resource_group_name', resource_group_name_type)

```

Comment:
i think this parameter shouldn't be removed

Reply:
Ohh this is just because I am using directive disable this method in list. I am testing whether it can work when I only disable one method in a command.